src/utils/jsonTransform.js

```diff
@@ -0,0 +1,64 @@
+import {isAffirmative} from './utils';
+
+export function to_registration_service_format(src) {
```

Comment:
Here I think we can define the function, and then call `export default ${fnname}`. We can always add additional exports as necessary.